torch/jit/annotations.py

```diff
@@ -106,14 +106,26 @@ def parse_type_line(type_line):
 def get_type_line(source):
     """Tries to find the line containing a comment with the type annotation."""
     lines = source.split('\n')
+    type_comment = '# type:'
+    type_lines = list(filter(lambda line: type_comment in line, lines))
 
-    type_line = None
-    for line in lines:
-        if '# type:' in line:
-            type_line = line.strip()
-            break
+    if len(type_lines) == 1:
+        return type_lines[0]
```

Comment:
Something like:
```
def foo(
   x, 
   y, 
   z # type: int)
```
Should be typed as (Tensor, Tensor, int). It looks like this wouldn't be properly handled at the moment

Reply:
Really that should be the same as doing something like this, which throws an error. Since this PR just constructs the long type line from the shorter pieces, the behavior here is fine

```python
@torch.jit.script
def test(x, y):
    # type: (int) -> Tensor
    return y + x
```